api/external/nodes/manager/manager.proto

```diff
@@ -169,62 +246,141 @@ service NodeManagerService {
 message ConnectResponse{}
 
 message Id {
+	option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+		json_schema: {
+			required: ["id"]
+		}
+		example: {value: '{"uuid": "cd3ad3d9-2776-4ef1-a904-4c229d1642ee"}'}
+	};
+	// UUID for the node manager.
 	string id = 1;
 }
 message Ids {
+	// List of node manager UUIDs.
 	repeated Id ids = 1;
 }
 
 message Query {
+	option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+		json_schema: {
+			required: ["filter_map"]
+			}
+		example: {value: '{"filter_map": [{"key": "manager_type","values":["aws-ec2"]}],"sort": "date_added"}'}
+	};
+	// Filters for the query: "manager_type".
```

Comment:
I'm not sure what this comment is supposed to convey. Is `manager_type` the only valid filter?

Reply:
yupp: https://github.com/chef/automate/blob/master/components/nodemanager-service/pgdb/node-managers.go#L337